.gitignore

```diff
@@ -0,0 +1,31 @@
+# Logs
```

Comment:
Can we copy the settings from https://github.com/GoogleChrome/chrome-extensions-samples/tree/main for consistency?

Reply:
Should I keep `.vscode` in the gitignore?

Reply:
If you need it, sure :)